uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -124,6 +131,7 @@ public static UiObject obtainUiObjectWithText(@NonNull final String text) {
      */
     public static UiObject obtainUiObjectWithResourceIdAndText(@NonNull final String resourceId,
                                                                @NonNull final String text) {
+        Logger.i(TAG, "Obtain an instance of an UiObject for the given resource Id and text..");
```

Comment:
let's also log the resource id in the log message

Reply:
added.